finality-aleph/src/party/manager/mod.rs

```diff
@@ -72,13 +70,11 @@ type CurrentNetworkType = SimpleNetwork<
     SessionSender<CurrentRmcNetworkData>,
 >;
 
-struct SubtasksParams<C, B, N, BE, JS>
+struct SubtasksParams<H, HB, N, JS>
 where
-    B: BlockT<Hash = BlockHash>,
-    B::Header: HeaderT<Number = BlockNumber> + UnverifiedHeader,
-    C: crate::ClientForAleph<B, BE> + Send + Sync + 'static,
-    BE: Backend<B> + 'static,
-    N: Network<VersionedNetworkData<B::Header>> + 'static,
+    H: Header,
+    HB: crate::block::HeaderBackend<H> + Send + Sync + 'static,
```

Comment:
Could you maybe just import it rather than fully qualifying here?

Reply:
Done